samples/excel/85-preview-apis/workbook-insert-external-worksheets.yaml

```
@@ -15,21 +15,26 @@ script:
 
             reader.onload = (event) => {
                 Excel.run((context) => {
-                    // strip off the metadata before the base64-encoded string
+                    // Remove the metadata before the base64-encoded string.
                     const startIndex = reader.result.toString().indexOf("base64,");
                     const workbookContents = reader.result.toString().substr(startIndex + 7);
-
-                    const sheets = context.workbook.worksheets;
-                    sheets.addFromBase64(
-                        workbookContents,
-                        null, // get all the worksheets 
-                        Excel.WorksheetPositionType.end // insert them after the current workbook's worksheets
-                        );
+                    
+                    // Retrieve the workbook.
+                    const workbook = context.workbook;
+                    
+                    // Set up the insert options. 
+                    var options = { 
+                        sheetNamesToInsert: [], // Insert all the worksheets from the source workbook.
+                        positionType: Excel.WorksheetPositionType.after, // Insert after the `relativeTo` sheet.
+                        relativeTo: "Sheet1" }; // A worksheet name. Use with `positionType` to specify insert location.
```

The other argument comments seem specific to the sample scenario, so consider framing this one like that too.
```suggestion
                        relativeTo: "Sheet1" }; // The sheet relative to which the other worksheets will be inserted.
```